docs/Demos.md

```diff
@@ -7,6 +7,196 @@ title: "React-admin demos"
 
 If you want to see what react-admin is capable of, or if you want to learn from apps built by seasoned react-admin developers, check out these demos.
 
+## Overview
+
+Discover our demos that show you the react-admin potential and help you familiarize with it.
+
+<style>
+    .demos-list {
+        display: grid;
+        grid-template-columns: repeat(2, 1fr);
+        grid-template-rows: repeat(3, 1fr);
+        grid-gap: 10px;
+    }
+
+    @media (max-width: 600px) {
+        .demos-list {
+            display: flex;
+            flex-wrap: wrap;
+            gap: 10px;
+        }
+    }
+
+    .card {
+        box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+        transition: 0.3s;
+        background-color: rgba(243, 246, 249, 0.4);
+        border: 1px solid rgba(0,0,0,0.25);
+        border-radius: 30px;
+        margin: 0;
+        display: flex;
+        flex-direction: column;
+    }
+
+    .card:hover {
+        box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
+    }
+
+    .card > img, .card > video {
+        border-radius: 15px;
+        width: 100%;
+    }
+
+    iframe {
+        width: calc(100% - 2rem);
+        height: calc(100% - 2rem);
+        border-radius: 15px;
+        margin: 1rem;
+    }
+
+    #player {
+        width: 100%;
+    }
+
+    .content-card {
+        padding: 2px;
+        margin-left: 1rem;
+        margin-right: 1rem;
+        text-decoration: none !important;
+    }
+
+    .title-card {
+        margin-bottom: 0.4rem !important;
+        font-size: 1.75em;
+        color: black;
+    }
+
+    .description-card {
+        color: rgb(100,100,100);
+    }
+
+    .card-footer {
+        flex-grow: 1;
+        display: flex;
+        flex-direction: column-reverse;
+    }
+
+    .links-container {
+        padding: 2px;
+        margin-left: calc(1rem - 10px);
+        margin-right: calc(1rem - 10px);
+        display: flex;
+        justify-content: space-between;
+    }
+
+    .link {
+        color: rgb(0, 127, 255) !important;
+        padding: 10px;
+        border-radius: 15px
+    }
+
+    .link:hover {
+        background-color: rgba(0, 127, 255, 0.07) !important;
+    }
+
+    .no-decoration, .link {
+        text-decoration: none !important;
+    }
+</style>
+
+<div class="demos-list">
+    <div class="card">
+        <iframe src="https://player.vimeo.com/video/474999017?autoplay=1&loop=1&muted=1" frameborder="0" allowfullsreen></iframe>
+        <div class="content-card">
+            <a href="#e-commerce" class="no-decoration">
```

Comment:
I'm a bit lost when I click the links anchor. I would have imagined dedicated pages for each sections instead, that explain further what they demonstrate. 
What do you think about it?

Reply:
I'm a bit worried about the number of pages this will create. won't it overload the navbar?

Reply:
For the record, I disagree with @adguernier , IMO the anchors are fine and we should keep them that way.
To be discussed with @fzaninotto then.